apm-agent-core/src/main/java/co/elastic/apm/agent/configuration/CoreConfiguration.java

```diff
@@ -318,6 +319,18 @@
             "of their duration.\n")
         .buildWithDefault(TimeDuration.of("0ms"));
 
+    private final ConfigurationOption<String> appendPackagesToBootDelagationProperty = ConfigurationOption.stringOption()
+        .key("boot_delegation_packages")
+        .configurationCategory(CORE_CATEGORY)
+        .description("A comma-separated list of packages to be appended to the boot delegation system property. \n" +
+            "If set with an empty string, nothing will be appended to the boot delegation system property.\n" +
+            "Values to set in known environments:\n\n" +
+            "Nexus:\n" +
+            "boot_delegation_packages=com.sun.*,javax.transaction,javax.transaction.*,javax.xml.crypto,javax.xml.crypto.*,sun.*,co.elastic.apm.agent.*\n\n" +
```

Comment:
```suggestion
            "`boot_delegation_packages=com.sun.*,javax.transaction,javax.transaction.*,javax.xml.crypto,javax.xml.crypto.*,sun.*,co.elastic.apm.agent.*`\n\n" +
```

Reply:
@felixbarny I wanted this to be a line that can be just uncommented from the generate property file